ncm/usb.go

```diff
@@ -29,6 +29,7 @@ var deviceLock = sync.Mutex{}
 var deviceCounter = 0
 
 func Start(c chan os.Signal) error {
+	go startPrometheus(13333)
```

Comment:
It would be nice to have this configurable via env or CLI args. Not crucial. 👍

Reply:
I think that makes sense. I'll add that.